src/DynamoCoreWpf/Views/PackageManager/Controls/SearchBoxControl.xaml.cs

```diff
@@ -16,28 +14,28 @@ namespace Dynamo.PackageManager.UI
     public partial class SearchBoxControl : UserControl
     {
         private DispatcherTimer delayTimer;
+        private string userSearchInput = ""; // Store user's input separately
+        private static int delayTime = 100;  // set delay for event 100ms
 
-        // set delay for event 500ms
-        private static int delayTime = 500;
         /// <summary>
         /// Constructor
         /// </summary>
         public SearchBoxControl()
         {
             InitializeComponent();
+
+            delayTimer = new DispatcherTimer();
+            delayTimer.Interval = TimeSpan.FromMilliseconds(delayTime);
+            delayTimer.Tick += DelayTimer_Tick;
```

Comment:
you should remove this handler somewhere.

Reply:
done